tests/test_services_v2.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from flow360.component.simulation import services
+
+
+@pytest.fixture(autouse=True)
+def change_test_dir(request, monkeypatch):
+    monkeypatch.chdir(request.fspath.dirname)
+
+
+def test_validate_service_empty():
+    params_data = {}
+
+    errors = services.validate_model(params_as_dict=params_data, unit_system_name="SI")
+
+    assert errors is None
+
+
+def test_validate_service():
+    params_data = {
```

Comment:
We will need to update these later because whether things are optional or not is not fully implemented or considered. But this is a very good base nonetheless.